pom.xml

```diff
@@ -750,6 +755,24 @@
                     </webApp>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.datanucleus</groupId>
+                <artifactId>datanucleus-maven-plugin</artifactId>
+                <version>${plugin.datanucleus.version}</version>
+                <configuration>
+                    <api>JDO</api>
+                    <verbose>true</verbose>
+                    <fork>false</fork>
+                </configuration>
+                <executions>
+                    <execution>
+                        <phase>process-classes</phase>
+                        <goals>
+                            <goal>enhance</goal>
+                        </goals>
+                    </execution>
+                </executions>
+            </plugin>
```

Comment:
What's the reason for including this directly?

Reply:
When using IntelliJ to compile everything, it wasn't enhancing the classes before adding this. Some of the changes in this PR are not exactly necessary and are purely for my own convenience, althouth I have tried to keep these changes to a minimum.